### ISSUE TITLE
[inductor] HuggingFace training failures tracker

### ISSUE THREAD

**Issue by desertfire** [pytorch/torchdynamo#779]
[Edited]
`python benchmarks/huggingface.py -d cuda --inductor --training --float32 --use-eval-mode`

Snapshot of Aug 22:

- [x] #964
  - BigBird
- [x] #840
  - CamemBert
- [x] #959 
  - AllenaiLongformerBase
  - GoogleFnet
- [x] #790
  - MBartForCausalLM
  - MBartForConditionalGeneration
- [ ] OOM Error, #822 : 
  - AlbertForMaskedLM
  - AlbertForQuestionAnswering
  - BartForCausalLM
  - DebertaForMaskedLM
  - DebertaForQuestionAnswering
  - DebertaV2ForMaskedLM
  - DebertaV2ForQuestionAnswering
  - DistilBertForMaskedLM
  - GPTNeoForCausalLM
  - GPTNeoForSequenceClassification
  - M2M100ForConditionalGeneration
  - MT5ForConditionalGeneration
  - PegasusForCausalLM
  - PegasusForConditionalGeneration
  - TrOCRForCausalLM
- [ ] Accuracy Error : 
  - BartForConditionalGeneration
  - MobileBertForMaskedLM
  - PLBartForConditionalGeneration

The following ones also fail with aot_nvfuser
- [ ] XGLMForCausalLM
- [ ] DebertaV2ForMaskedLM
- [ ] GPTNeoForCausalLM
- [ ] GPTNeoForSequenceClassification

**Comment by desertfire** [comment]
Combine with https://github.com/pytorch/torchdynamo/issues/511
